services/web/server/src/simcore_service_webserver/director_v2_abc.py

```diff
@@ -0,0 +1,54 @@
+from abc import ABC, abstractmethod
+from typing import List, Optional, Tuple
+
+from aiohttp import web
+from models_library.projects import ProjectID
+
+CommitID = int
+
+
+class AbstractProjectRunPolicy(ABC):
```

Comment:
I am not sure in that case that having Abstract in the name of the class makes it easier to understand.
It's a ProjectRunPolicy, the fact that it is abstract is a detail of the implementation right? From the viewpoint of the director-v2 it is the actual policy right?

Reply:
I try to follow the same approach as pydantic with e.g. ``BaseModel``  which keeps "base" in the class to say "you need to inherit" ... i can change it though